hackcon-2015/README.md

```diff
@@ -0,0 +1,27 @@
+# hackcon15 write-ups
+
+* [Website](http://hackcon.in/)
+* [Scoreboard](http://hackcon.in/scoreboard)
+
```

Comment:
A static copy of the leaderboard would work? In the repo?
On Aug 17, 2015 3:36 PM, "Tummala Dhanvi" <notifications@github.com> wrote:

> In hackcon-2015/README.md
> <https://github.com/ctfs/write-ups-2015/pull/312#discussion_r37173625>:
>
> > @@ -0,0 +1,27 @@
> > +# hackcon15 write-ups
> > +
> > +* [Website](http://hackcon.in/)
> > +* [Scoreboard](http://hackcon.in/scoreboard)
> > +
>
> give us a permanent score board
>
> ie it should be valid even after you guys put down the server or conduct a
> new one next year
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ctfs/write-ups-2015/pull/312/files#r37173625>.
>